UnityProject/Assets/Scripts/ChemistryComponents/ReagentContainer.cs

```diff
@@ -458,18 +467,9 @@ private void ExamineContents()
 				Chat.AddExamineMsgToClient($"The {gameObject.ExpensiveName()} is empty.");
```

Comment:
This is the server running client side only chat functions. Could you change them to AddExamineMsgFromServer instead